doc/classes/HTTPRequest.xml

```diff
@@ -57,11 +57,11 @@
 		    // Perform a POST request. The URL below returns JSON as of writing.
 		    // Note: Don't make simultaneous requests using a single HTTPRequest node.
 		    // The snippet below is provided for reference only.
-		    string body = new Json().Stringify(new Godot.Collections.Dictionary
+		    string body = Json.Stringify(new Godot.Collections.Dictionary
```

Comment:
The GDScript example is also outdated, it uses the static `stringify` method on a JSON instance.